### PR TITLE
Marketing Launch Plan: Web analytics 

### PR DESCRIPTION
**Pull request by joethreepwood**
I like making checklists.

### CODE REVIEW

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
+- [ ] Blog + Changelog announcement at launch - @joethreepwood
+- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital 
+
+## Bigger ideas
+
+Ideas for other, weirder things go here.
```
Comment (robbie-c) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:55]: Chatted with @ivanagas about potentially live-coding some Web Analytics features and putting an edited video on youtube.

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital 
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
+- [ ] Limited edition merch drop for users in the beta: "I was in PostHog's beta and all I got was this awesome t-shirt!"
+- [ ] Web Analytics launch on Product Hunt
```
[the remark this answers is not in the document]
Comment (joethreepwood) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:58]: LETS DO IT

[the remark this answers is not in the document]
Comment (charlescook-ph) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:58]: The quality of audience is so bad on PH I'd love to not spend time here tbh. Even though I think we could do really well, I think we would just generate a ton of WoM with people who don't want to spend money but have lots of support questions.

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [ ] Article comparing performance impact of PostHog vs other web analytics tools
+- [ ] Web analytics comparison content for: Plausible, Fathom
+- [ ] Video tutorial about how to use web analytics
+- [ ] Give Theo information as to how we built Web Analytics for a video
```
Comment (ivanagas) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:62]: Related idea, we do something with [Morning Makers](https://morningmakershow.com/) on Twitter. Guide to web analytics for indie hackers?

File: requests-for-comments/2024-02-13-web-analytics-launch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joethreepwood) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:58]: I don't feel we've fully proven this tbh, and I'd love to give it another try. I feel like we missed a trick with this on surveys for the same reason. 

Maybe we compromise and look out to some new, untested communities -- Indiehackers or Betalist?

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- [ ] Blog + Changelog announcement at launch - @joethreepwood
+- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital 
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:56]: @robbie-c @ivanagas What would you think about adding a tool/doc about how to migrate from GA4 to PostHog? 

I'm unclear on the process here, but I imagine anything we do to support migration of data is going to help encourage adoption.

Comment (joethreepwood) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:56]: Amplitude and Mixpanel have this in their docs
https://www.docs.developers.amplitude.com/data/tutorials/migrating-from-google-analytics/
https://docs.mixpanel.com/docs/migration/google-analytics

File: requests-for-comments/2024-02-13-web-analytics-launch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (charlescook-ph) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:58]: I'm happy to try Product Hunt again and be proven wrong here. So long as we're looking further down the funnel at ICPs/billing subscribed, and not just the vanity stuff. Maybe we give it a proper go and then either decide it was a success to repeat, or not to try it again?

It looks like if you hit top 3, [you should expect 2-3k views and 50-100 signups](https://dev-mkt.slack.com/archives/C057KGB5UBZ/p1700832241814809?thread_ts=1700828246.089149&cid=C057KGB5UBZ) - which is far lower than front page of Hacker News territory. 

Also worth bearing in mind PH has changed quite a lot and [has become more pay-to-win](https://dev.to/wasp/product-hunt-has-become-pay-to-win-but-you-should-still-use-it-to-launch-your-product-508d), so we should factor that into the plan.

(I think it's pretty clear I'm down on PH, but will disagree and commit here - would rather do that than try a much smaller community.)

File: requests-for-comments/2024-02-13-web-analytics-launch.md (hunk 4 above)
Comment (robbie-c) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:56]: Doc yes, tool maybe. Internal thread on GA4 import https://posthog.slack.com/archives/C05LJK1N3CP/p1707910732916289

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
+- [ ] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
+
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
```
Comment (raquelmsmith) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:33]: I think it'll be part of product analytics, see https://github.com/PostHog/product-internal/pull/566

Comment (joethreepwood) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:33]: As in it'll be included with product analytics, or some sort of sub-choice from there? 

I'd like to have a web analytics path in the onboarding emails, but in order to do that I'd ideally need users to make an explicit choice to focus on it.

Comment (raquelmsmith) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:33]: tbd, will update

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+Pricing is TBC, but web analytics will have usage-based pricing. 
+
+## Site updates
+
+Basic stuff across the site that'll need to be done. 
+
+- [ ] Update documentation - @andyvan https://github.com/PostHog/posthog.com/issues/7736
+- [ ] Create a product page - @corywaitlo https://github.com/PostHog/posthog.com/pull/8001
+- [ ] Web analytics case study - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:19]:
```suggestion
- [x] Web analytics case study - @joethreepwood
```

**Comment by robbie-c** [comment]
So this is probably implicit and known by everyone in this thread, but to make it explicit: we are delaying the web analytics to prioritise the personless events launch.

**Comment by corywatilo** [comment]
Related: [artwork thread for launch announcement](https://github.com/PostHog/posthog.com/issues/9265)

**Comment by robbie-c** [comment]
> So this is probably implicit and known by everyone in this thread, but to make it explicit: we are delaying the web analytics to prioritise the personless events launch.

We've removed the beta tag and can do the launch when we're ready

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- [ ] Create a product page - @corywaitlo https://github.com/PostHog/posthog.com/pull/8001
+- [x] Web analytics case study - @joethreepwood
+- [ ] Remove beta tags from homepage slider, docs, etc - @joethreepwood
+- [ ] Add Web analytics links to footer - @joethreepwood
+- [ ] Update `/blog/ga4-alternatives` with a CTA
+- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
+- [ ] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:27]:
```suggestion
- [x] Update documentation - @andyvan https://github.com/PostHog/posthog.com/issues/7736
- [x] Create a product page - @corywaitlo https://github.com/PostHog/posthog.com/pull/8001
- [x] Web analytics case study - @joethreepwood
- [x] Remove beta tags from homepage slider, docs, etc - @joethreepwood
- [x] Add Web analytics links to footer - @joethreepwood
- [ ] Update `/blog/ga4-alternatives` with a CTA
- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
- [x] Create a Web Analytics section for `/questions`
- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
+- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
+- [ ] Email announcement to beta users _ahead of launch_ - @joethreepwood (preferential pricing?)
+- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
+- [ ] Blog + Changelog announcement at launch - @joethreepwood
+- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:51]:
```suggestion
- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
- [ ] Email announcement to beta users _ahead of launch_ - @joethreepwood (preferential pricing?)
- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
- [ ] Blog + Changelog announcement at launch - @joethreepwood
- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
- [ ] Pylon announcement to all current paying users - @joethreepwood
- [x] New key art for use in blogs, emails, etc - @lottiecoxon
- [ ] New web analytics ads run by Hey Digital - @andyvan-ph 
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
+- [ ] Limited edition merch drop for users in the beta: "I was in PostHog's beta and all I got was this awesome t-shirt!"
+- [ ] Web Analytics launch on Product Hunt
+- [ ] Article comparing performance impact of PostHog vs other web analytics tools
+- [ ] Web analytics comparison content for: Plausible, Fathom
+- [ ] Video tutorial about how to use web analytics
+- [ ] Give Theo information as to how we built Web Analytics for a video
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:63]:
```suggestion
- [ ] Article comparing performance impact of PostHog vs other web analytics tools
- [ ] Web analytics comparison content for: Plausible, Fathom
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital 
+- [ ] Invite YC teams who don't use us on our marketing site to try us out
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
+- [ ] Limited edition merch drop for users in the beta: "I was in PostHog's beta and all I got was this awesome t-shirt!"
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:58]:
```suggestion
```

**Review by joethreepwood — COMMENTED** [review]
Updates

File: requests-for-comments/2024-02-13-web-analytics-launch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joethreepwood) [reply requests-for-comments/2024-02-13-web-analytics-launch.md:58]: On reflection, I'm starting to lean no on this -- I'll park it for now, just as I've come to agree with Charles that the effort for PH would be pretty high compared to the win.

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [x] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital - @andyvan-ph 
+- [ ] Invite YC teams who don't use us on our marketing site to try us out
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
+- [ ] Web Analytics launch on Product Hunt
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:58]:
```suggestion
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+- [ ] Invite YC teams who don't use us on our marketing site to try us out
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
+- [ ] Article comparing performance impact of PostHog vs other web analytics tools
+- [ ] Web analytics comparison content for: Plausible, Fathom
+- [ ] Announce via newsletter sponsorships 
+- [ ] Approach Jason Packer about contributing to the newsletter/blog
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:61]:
```suggestion
- [ ] Approach Jason Packer about contributing to the newsletter/blog
- [ ] Unique hedgehog mode skin design and implementation - @lottiecoxon & @robbie-c 
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+- [x] Create a product page - @corywaitlo https://github.com/PostHog/posthog.com/pull/8001
+- [x] Web analytics case study - @joethreepwood
+- [x] Remove beta tags from homepage slider, docs, etc - @joethreepwood
+- [x] Add Web analytics links to footer - @joethreepwood
+- [ ] Update `/blog/ga4-alternatives` with a CTA
+- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
+- [x] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:27]:
```suggestion
- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
- [x] Migration guide: GA > PostHog - Ian
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+
+- [x] Update documentation - @andyvan https://github.com/PostHog/posthog.com/issues/7736
+- [x] Create a product page - @corywaitlo https://github.com/PostHog/posthog.com/pull/8001
+- [x] Web analytics case study - @joethreepwood
+- [x] Remove beta tags from homepage slider, docs, etc - @joethreepwood
+- [x] Add Web analytics links to footer - @joethreepwood
+- [ ] Update `/blog/ga4-alternatives` with a CTA
+- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:25]:
```suggestion
- [x] Update `/blog/ga4-alternatives` with a CTA - @joethreepwood
- [x] Update `/blog/track-your-website-with-posthog` with a CTA - @joethreepwood
- [x] Update `/blog/google-analytics-to-posthog` with a CTA - @joethreepwood
- [x] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA - @joethreepwood
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+- [x] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
+- [x] Migration guide: GA > PostHog - Ian
+
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
+- [ ] Add Web Analytics path as an email onboarding option - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:35]:
```suggestion
- [x] Add Web Analytics path as an email onboarding option - @joethreepwood
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
+- [x] Migration guide: GA > PostHog - Ian
+
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
+- [x] Add Web Analytics path as an email onboarding option - @joethreepwood
+- [ ] Add Web Analytics to G2, Bookface, etc - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:36]:
```suggestion
- [x] Add Web Analytics to G2 - @joethreepwood
- [x] Add Web Analytics to Bookface - @joethreepwood
- [x] Add Web Analytics to LinkedIn profile - @joethreepwood
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
+- [x] Add Web Analytics path as an email onboarding option - @joethreepwood
+- [x] Add Web Analytics to G2 - @joethreepwood
+- [x] Add Web Analytics to Bookface - @joethreepwood
+- [x] Add Web Analytics to LinkedIn profile - @joethreepwood
+- [ ] Update PostHog README.md with information - @joethreepwood
+- [ ] Add PostHog to any relevant new G2 categories
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:40]:
```suggestion
- [x] Add PostHog to any relevant new G2 categories
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
+- [x] Add Web Analytics path as an email onboarding option - @joethreepwood
+- [x] Add Web Analytics to G2 - @joethreepwood
+- [x] Add Web Analytics to Bookface - @joethreepwood
+- [x] Add Web Analytics to LinkedIn profile - @joethreepwood
+- [ ] Update PostHog README.md with information - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:39]:
```suggestion
- [x] Update PostHog README.md with information - @joethreepwood
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+- [x] Update PostHog README.md with information - @joethreepwood
+- [x] Add PostHog to any relevant new G2 categories
+
+## Default launch activities
+
+Basic things we always do. 
+
+- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
+- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
+- [ ] Email announcement to beta users _ahead of launch_ - @joethreepwood (preferential pricing?)
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:48]:
```suggestion
- [ ] Email announcement to beta users at launch with special reward - @joethreepwood 
```

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+
+## Default launch activities
+
+Basic things we always do. 
+
+- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
+- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
+- [ ] Email announcement to beta users at launch with special reward - @joethreepwood 
+- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
+- [ ] Blog + Changelog announcement at launch - @joethreepwood
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:50]:
```suggestion
- [ ] Blog / Changelog announcement at launch - @joethreepwood
```

**Comment by joethreepwood** [comment]
We're setting Thurs 12 Sept as the launch date, FYI

File: requests-for-comments/2024-02-13-web-analytics-launch.md [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
+- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
+- [ ] Email announcement to beta users at launch with special reward - @joethreepwood 
+- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
+- [ ] Blog / Changelog announcement at launch - @joethreepwood
+- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [x] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital - @andyvan-ph 
+- [ ] Invite YC teams who don't use us on our marketing site to try us out
```
Comment (joethreepwood) [inline requests-for-comments/2024-02-13-web-analytics-launch.md:55]:
```suggestion
- [x] In-app announcement using surveys/notifications for all users - @joethreepwood
- [x] Email announcement to beta users at launch with special reward - @joethreepwood 
- [x] Email announcement to all remaining users _at launch_ - @joethreepwood
- [x] Blog / Changelog announcement at launch - @joethreepwood
- [x] Social announcements on Twitter + LinkedIn - @joethreepwood
- [x] Pylon announcement to all current paying users - @joethreepwood
- [x] New key art for use in blogs, emails, etc - @lottiecoxon
- [ ] New web analytics ads run by Hey Digital - @andyvan-ph 
- [x] Invite YC teams who don't use us on our marketing site to try us out
```

**Pull request merged**